core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -580,6 +580,52 @@ class TransactionsTest extends KafkaServerTestHarness {
     assertTrue(transactionalRecords.isEmpty)
   }
 
+  @Test
+  def testProducerUsableAfterTxnTimeOutAbortExcetion(): Unit = {
```

Comment:
Could we use the same template and reproduce the scenario where user doesn't call abortTxn and immediately resume working?

Reply:
ok. The reverse verification is added.